src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,153 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //----------------------------------------------------------------------------
+  // Container
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-01; // allow space for focus styles
+  }
+
+  //----------------------------------------------------------------------------
+  // Table title text
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //----------------------------------------------------------------------------
+  // Data table
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 thead {
+    @include type-style('heading-01');
+    background-color: $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 tbody {
+    @include type-style('body-short-01');
+    background-color: $ui-01;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tr {
+    border: none;
+    height: $layout-04;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover {
+    background: $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover td {
+    color: $text-01;
+    background: $hover-field;
+    border-bottom: 1px solid $hover-field;
+    border-top: 1px solid $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 th,
+  .#{$prefix}--data-table-v2 td {
+    padding-left: $spacing-04;
+    padding-right: $spacing-04;
+    vertical-align: top;
+    text-align: left;
+  }
+
+  .#{$prefix}--data-table-v2 th {
+    color: $text-01;
+    background-color: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+    padding-left: $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2 .#{$prefix}--table-header-label {
+    display: block;
+    padding: rem(14px) 0;
+  }
+
+  .#{$prefix}--data-table-v2 td {
+    background: $ui-01;
+    color: $text-02;
+    border-top: 1px solid $ui-01;
+    border-bottom: 1px solid $ui-03;
+    padding: rem(14px) $spacing-04;
```

Comment:
Instead of using padding to create vertical centering, is it possible to use `vertical-align: middle`?

Reply:
So in v9 rows have `vertical-align: middle` but in v10 design wants them to be `vertical-align: top` if there's text wrapping. So if there's no overflow, it looks like it's centered vertically, but if it overflows it's not centered vertically, it just has some top padding. Not sure if that explanation makes sense.

See first row here as an example. When there's really long text that wraps, other text that isn't as long should still be aligned at the top.

<img width="729" alt="Screen Shot 2019-03-20 at 4 40 17 PM" src="https://user-images.githubusercontent.com/32556167/54721009-e095c000-4b2e-11e9-8662-c37c0cfe394f.png">

Reply:
Makes sense!